### ISSUE TITLE
Screamer during SMB right after init

### ISSUE THREAD

**Issue by danielhers** [AAPS-Omnipod/AndroidAPS-OLD#120]
TIME OF EVENT HERE: 09:19
ISSUE DESCRIPTION: screamer a minute after initializing; while delivering smb

AndroidAPS 2.7-omnipod-0.3
Build: EROS_0.3-2020.03.14-21:01
Remote: git@github.com:danielhers/AndroidAPS.git
Flavor: fullrelease
Nightscout version: 13.0.2-dev
Engineering mode enabled

**Comment by danielhers** [comment]
[AndroidAPS_LOG_1587281353781.log.zip](https://github.com/AAPS-Omnipod/AndroidAPS/files/4498687/AndroidAPS_LOG_1587281353781.log.zip)

**Comment by bartsopers** [comment]
These logs don't contain any information about the Fault Event. Can you look for a log file that contains "PodFaultException"?

**Comment by danielhers** [comment]
[AndroidAPS._2020-04-19_19-08-46_.0.zip](https://github.com/AAPS-Omnipod/AndroidAPS/files/4501445/AndroidAPS._2020-04-19_19-08-46_.0.zip)
Right, sorry. Here is the one after.

**Comment by danielhers** [comment]
Looks like there was no "PodFaultException" in this event, but the pod started screaming and became unusable anyway. The log I sent does contain the event time. Possible related entries:

- 09:19:21.592: `isDeviceReachable. Response is invalid !`
- 09:19:25.055: `writeToData: RileyLink reports timeout`
- 09:19:31.69: `Caught exception in executeAndVerify. Verifying command by using cancel none command to verify nonce`
- 09:19:38.323: `Command unresolved (UNCERTAIN_FAILURE)`

After that I tried to deactivate the pod but that failed (09:21:35.611), so I had to reset pod (09:21:59.729) and then I could init and use another pod with no problem.
